rope/contrib/autoimport/sqlite.py

```diff
@@ -434,11 +441,14 @@ def clear_cache(self):
         """
         with self.connection:
             self._execute(models.Name.objects.drop_table())
+            self._execute(models.Alias.objects.drop_table())
             self._execute(models.Package.objects.drop_table())
             self._execute(models.Metadata.objects.drop_table())
             models.Name.create_table(self.connection)
+            models.Alias.create_table(self.connection)
             models.Package.create_table(self.connection)
             models.Metadata.create_table(self.connection)
+            self.add_aliases(self.project.prefs.import_aliases)
             data = (
```

Comment:
So if I understand this correctly, this will add the aliases into the database only when the database is created. IIUC, this would need to depend on the database being re-created when preference changes.

Reply:
You're right. I didn't look at the different ways that prefs can change, we could add a method to clear the aliases table and reset it and invoke that when the prefs are updated.